contracts/src/bridge/Inbox.sol

```diff
@@ -115,12 +115,6 @@ contract Inbox is DelegateCallAware, PausableUpgradeable, IInbox {
 
     /// @inheritdoc IInbox
     function postUpgradeInit(IBridge _bridge) external onlyDelegated onlyProxyOwner {
-        uint8 slotsToWipe = 3;
-        for (uint8 i = 0; i < slotsToWipe; i++) {
-            assembly {
-                sstore(i, 0)
-            }
-        }
         allowListEnabled = false;
         bridge = _bridge;
```

Comment:
Is there a reason to leave the rest of this function? Can't hurt, but it's been run on rinkeby and mainnet already so I don't think we want to run it again

Reply:
removed in fc947df3